view/base.js

```diff
@@ -329,6 +331,7 @@ if ( typeof kintInitialized === 'undefined' ) {
 	}, false);
 
 	// keyboard navigation
+	var j_toggles = 1;
```

Comment:
Nitpick: j_toggles is an undescriptive variable name (I know, the file is not that readable already), but it a) indicates it's a counter just by it not having a boolean value b) `navStartedByJ` or something similar would adhere to camelCase convention of the whole file and describe the purpose of the variable better